#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_get_rid_of_utillinux-mountbinary.dpatch by  <mika@grml.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Get rid of util-linux mount binary

@DPATCH@

diff -r 9c83db37b8d9 hooks/live
--- a/hooks/live	Sun Jul 20 19:19:41 2008 +0200
+++ b/hooks/live	Fri Aug 01 16:07:05 2008 +0200
@@ -74,11 +74,11 @@
 fi
 
 # Filesystem: cifs
-if [ -x /sbin/mount.cifs ]
-then
-	copy_exec /sbin/mount.cifs /sbin
-        manual_add_modules cifs
-fi
+#if [ -x /sbin/mount.cifs ]
+#then
+#	copy_exec /sbin/mount.cifs /sbin
+#        manual_add_modules cifs
+#fi
 
 # Filesystem: ext3
 manual_add_modules ext3
